docs/admin/security/audit-logs.md

```diff
@@ -26,9 +26,9 @@ We track the following resources:
+| WorkspaceTable<br><i></i>                                | <table><thead><tr><th>Field</th><th>Tracked</th></tr></thead><tbody><tr><td>automatic_updates</td><td>true</td></tr><tr><td>autostart_schedule</td><td>true</td></tr><tr><td>created_at</td><td>false</td></tr><tr><td>deleted</td><td>false</td></tr><tr><td>deleting_at</td><td>true</td></tr><tr><td>dormant_at</td><td>true</td></tr><tr><td>favorite</td><td>true</td></tr><tr><td>id</td><td>true</td></tr><tr><td>last_used_at</td><td>false</td></tr><tr><td>name</td><td>true</td></tr><tr><td>organization_id</td><td>false</td></tr><tr><td>owner_id</td><td>true</td></tr><tr><td>template_id</td><td>true</td></tr><tr><td>ttl</td><td>true</td></tr><tr><td>updated_at</td><td>false</td></tr></tbody></table>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Can we keep the old resource name? I don't think admins care if there is a difference between a change to a `WorkspaceTable` or a change to a `Workspace`?

Reply:
The type is still `workspace` to admins: https://github.com/coder/coder/blob/b05b00c797506ed6a0512b9f164f7ee7d71fece5/coderd/audit/request.go#L184-L185

It is just `WorkspaceTable` in Go